nodebuilder/store.go

```diff
@@ -186,6 +186,19 @@ func keysPath(base string) string {
 	return filepath.Join(base, "keys")
 }
 
+func blocksPath(base string) string {
+	return filepath.Join(base, "blocks")
+}
+
+func transientsPath(base string) string {
+	// we don't actually use the transients directory anymore, but it could be populated from previous versions.
```

Comment:
should you make an issue to remove this later?

Reply:
yes